.travis.yml

```diff
@@ -38,9 +37,7 @@ env:
 # linux-tools on Travis is a complicated issue
 install:
 - sudo apt-get install -y -qq linux-tools-`uname -r`
-# - sudo curl -sL https://github.com/ocaml/opam/releases/download/1.2.2/opam-1.2.2-x86_64-Linux -o /usr/bin/opam
-# - sudo chmod 755 /usr/bin/opam
-# - sudo curl -sL https://github.com/ocaml/opam/releases/download/2.0.1/opam-2.0.1-x86_64-linux -o /usr/bin/opam
+- sudo curl -sL https://github.com/ocaml/opam/releases/download/2.0.2/opam-2.0.2-x86_64-linux -o /usr/bin/opam
```

Comment:
We should stop telling apt to install opam. linux-tools is also redundant with the apt key.

Reply:
Ok for opam, however are you sure linux-tools-generic is redundant? At least on bionic the dep is not there.

Reply:
I'm confused, we install linux-tools-generic by travis and linux-tools-`uname -r` by command line, why do we need to do both?

Reply:
One is the kernel part and the other is the user space part. We need both as neither of them depend on the other, and the `apt` plugin doesn't allow us to grab the right kernel driver.

Anyways perf doesn't work on Travis.